examples/network_driver/load_config/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/scrapli/scrapligo/driver/base"
+	"github.com/scrapli/scrapligo/driver/core"
+	"github.com/scrapli/scrapligo/cfg"
+)
+
+//Pushes config from a file to a file on device and
+//loads it to the device. Cleans up afterwords.
+func main() {
+	d, err := core.NewCoreDriver(
+		"hostname",
+		"juniper_junos",
+		base.WithPort(22),
```

Comment:
is this needed?

Reply:
not needed, but cant hurt to have in example to show folks I suppose!

Reply:
@hellt did you mean Port? No, not needed.
Did you mean is NewCoreDriver() call needed? I think we do need a driver for NewCfgDriver() factory.